frontend/src/core/hotkeys/hotkeys.ts

```diff
@@ -100,6 +100,11 @@ const DEFAULT_HOT_KEY = {
   },
 
   // Editing Cells
+  "cell.name": {
+    name: "Name",
+    group: "Editing",
+    key: "Mod-Shift-l",
```

Comment:
lets change this to `Ctrl-n` since i think `Mod-Shift-l` doesn't work

Reply:
`Ctrl-n` doesn't work

Reply:
hmm is there a key that works for you. that one does for me